pkg/runner/local_docker.go

```diff
@@ -285,7 +285,7 @@ func (r *LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow *rp
 
 	if !cfg.KeepContainers {
 		defer func() {
-			_ = deleteContainers(cli, ow, containers)
+			_ = deleteContainers(cli, log, containers)
```

Comment:
Passing `log`, instead of `ow`, so that we have `run_id` as part of the log messages inside `deleteContainers`.